strings/power_for_large_numbers_represented_as_strings.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * \file
+ * \brief The gfg
+ *modulo-power-for-large-numbers-represented-as-strings
+ */
+#include <iostream> 
+using namespace std; 
+#define ll long long int 
+const ll MOD = 1e9 + 7; 
+// Returns modulo exponentiation for two numbers 
+// represented as long long int. It is used by 
+// powerStrings(). Its complexity is log(n) 
+ll powerLL(ll x, ll n) { 
+	ll result = 1; 
+	while (n)
+	 { 
+		if (n & 1) 
+			result = result * x % MOD; 
+		n = n / 2; 
+		x = x * x % MOD; 
+	} 
+	return result; 
+} 
+// Returns modulo exponentiation for two numbers 

```

Comment:
Same in other parts of the code.
Please use a comment structure like:

```c++
/**
 * Something here
 * Something else here
 */
```
**except** inside functions.